### ISSUE TITLE
mypy_boto3_dynamodb fails on import

### ISSUE THREAD

**Issue by sgiango** [youtype/mypy_boto3_builder#192]
**Describe the bug**
mypy_boto3_dynamodb fails on import

**To Reproduce**

1. Install `boto3 mypy_boto3_dynamodb`

```python
import mypy_boto3_dynamodb

...
```

**Actual output**

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/dev/testl/venv/lib/python3.8/site-packages/mypy_boto3_dynamodb/__init__.py", line 37, in <module>
    from .client import DynamoDBClient
  File "~/dev/testl/venv/lib/python3.8/site-packages/mypy_boto3_dynamodb/client.py", line 39, in <module>
    from .paginator import (
  File "~/dev/testl/venv/lib/python3.8/site-packages/mypy_boto3_dynamodb/paginator.py", line 43, in <module>
    from .type_defs import (
  File "~/dev/testl/venv/lib/python3.8/site-packages/mypy_boto3_dynamodb/type_defs.py", line 423, in <module>
    "SS": Sequence[str],
TypeError: 'ABCMeta' object is not subscriptable
```

**Expected output**

```
...
```

**Additional context**
mypy-boto3-dynamodb==1.26.97

**Comment by vemel** [comment]
Thank you for the report. SHould be fixed in `boto3-stubs 1.26.96.post1`. Please update and let me know if everything works as expected.

**Comment by sgiango** [comment]
thanks, working now
